src/gui/properties/speedplotview.h

```diff
@@ -0,0 +1,89 @@
+#ifndef SPEEDPLOTVIEW_H
+#define SPEEDPLOTVIEW_H
+
+#include <QGraphicsView>
+#include <QPen>
+#include <QQueue>
+#include <QMap>
+
+class QSpeedPlotView : public QGraphicsView
```

Comment:
To avoid possible confusion with the Qt classes rename the class to `SpeedPlotView`